templates/NOTES.txt

```diff
@@ -70,3 +70,23 @@ Please see `README.md` for examples.
 
 {{- end }}
 {{- end }}
+{{- if .Values.image -}}
+*******************
```

Comment:
```suggestion

*******************
```

A line, otherwise it looks like this:

```console
Please see `README.md` for examples.*******************
******WARNING******
*******************
```